matrix-sdk-android/build.gradle

```diff
@@ -130,6 +130,8 @@ dependencies {
 
     // Database
     implementation 'com.github.Zhuinden:realm-monarchy:0.7.1'
+    testImplementation libs.rx.rxKotlin
```

Comment:
in order to mock the realm instance we need to include a test dependency on `rx` as [realm internally uses it](https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/java/io/realm/Realm.java#L49) and `monarchy` doesn't include `rx` in its transitive dependencies

Reply:
Can you add the comment in the code? Also maybe move this line next to the other `testImplementation` items (I have no strong opinion on this last remark)?